pkg/client/unversioned/remotecommand/v3.go

```diff
@@ -108,3 +110,34 @@ func (e *streamProtocolV3) stream(conn streamCreator) error {
 	// waits for errorStream to finish reading with an error or nil
 	return <-errorChan
 }
+
+type decodedExitError struct {
+	remotecommand.ExitError
+}
+
+func (e *decodedExitError) String() string {
+	return e.ExitError.Message
+}
+
+func (e *decodedExitError) Error() string {
+	return e.ExitError.Message
+}
+
+func (e *decodedExitError) Exited() bool {
+	return e.ExitError.Exited
+}
+
+func (e *decodedExitError) ExitStatus() int {
```

Comment:
It's part of the `util.exec.ExitError` interface we are implementing here. I have added a line to make this explicit.